### PR TITLE
[BUGFIX] Correct patrols that give background

### LINKED ISSUE #3180: [BUG] Stolen apprentice has wrong background
### Type of bug

Code - general bugs related to how the game runs and responds to you.

### Description

Had the patrol where another clan's apprentice asks to join. Looked at the cat's bio after they joined, and their background is wrong. I think it has to do with them having a kittypet prefix?

### Grade

Unimportant - This bug has little to no impact on play.

### Steps to reproduce the behavior

I haven't tried to reproduce it, but it happened with the patrol that lets you steal other clan's apprentices.

### Patrol ID + Outcome Print (if applicable)

_No response_

### Commit # or Game Version Number

0.12.2

### Additional context

_No response_

### Screenshots

Screenshot (919)

### PR DESCRIPTION
**Pull request by rustykitty**
## About The Pull Request
Fixes the patrol attributes so that backstories/events with new cats work correctly.
Specifically, I code to actually pull a random backstory when we override it, then I noticed that some events' backstory attributes separated backstories by a comma and space instead of just a comma, so I added that in as well.

## Linked Issues
Fixes #3180, #3200, #3203

## Proof of Testing
Screenshot 2024-12-19 at 6 38 00 AM
Screenshot 2024-12-19 at 6 38 10 AM

## Changelog/Credits
Cats found on patrol or in event now have backstories assigned correctly

### CODE REVIEW

**Comment by scribblecrumb** [comment]
This should be aimed at the 0.12.0 branch

File: scripts/screens/MakeClanScreen.py
Comment (j-gynn) [inline scripts/screens/MakeClanScreen.py:1]: Why is this change in this PR? I think it ought to be in `release-0.12.0` anyway, but...

**Review by j-gynn — DISMISSED** [review]
One change seems like it doesn't belong in this PR?

File: scripts/screens/MakeClanScreen.py
Comment (rustykitty) [reply scripts/screens/MakeClanScreen.py:1]: oops, I might have rebased incorrectly

Comment (rustykitty) [reply scripts/screens/MakeClanScreen.py:1]: should be fixed now
image

**Review by scribblecrumb — APPROVED** [review]

**Pull request merged**